common/rfb/CSecurityTLS.cxx

```diff
@@ -56,6 +56,12 @@ StringParameter CSecurityTLS::X509CA("X509CA", "X509 CA certificate",
 StringParameter CSecurityTLS::X509CRL("X509CRL", "X509 CRL file",
                                      configdirfn("x509_crl.pem"),
                                      ConfViewer);
+StringParameter CSecurityTLS::X509CERT("X509CERT", "X509 client certificate",
```

Comment:
The other two are all upper case because they are abbreviations. So please use `X509Cert` here.